src/Tizen.NUI/src/public/WebView/WebSettings.cs

```diff
@@ -522,7 +523,7 @@ public void EnableExtraFeature(string str, bool tag)
         ///  Enable extra feature
```

Comment:
Enable -> Enables the given extra feature

Reply:
Done.
I updated it in another patch https://github.com/Samsung/TizenFX/pull/3520